docs/guides/storybook-npm-setup.md

```diff
@@ -0,0 +1,155 @@
+ ```
+
+see [`webpack.config.js`](./webpack.config.js) for more details, and these two blog posts for background.
+- [How to package your React Component for distribution via NPM](https://itnext.io/how-to-package-your-react-component-for-distribution-via-npm-d32d4bf71b4f) - good for context but didn't follow this one
+- [React + npm - How to Publish a React Component to npm](http://jasonwatmore.com/post/2018/04/14/react-npm-how-to-publish-a-react-component-to-npm) - followed this one + [webpack docs](https://webpack.js.org/concepts)
+
+## import only certain components
+Using [Webpack code splitting entry points](https://webpack.js.org/guides/code-splitting/#entry-points) as well as exporting from `packages/index.js` for max flexibility. 
+
+As explained in the README you can now do both
+
+```js
+import TimedTextEditor from '@bbc/react-transcript-editor/TimedTextEditor';
+```
+
+```js
+import { TimedTextEditor } from '@bbc/react-transcript-editor';
+```
+
+The difference is that 
+
+> However if you are not using `TranscriptEditor` it is recommended to follow the second option and import individual components like: `@bbc/react-transcript-editor/TimedTextEditor` rather than the entire library. Doing so pulls in only the specific components that you use, which can significantly reduce the amount of code you end up sending to the client. (Similarly to how [`react-bootstrap`](https://react-bootstrap.github.io/getting-started/introduction) works)
+
+
+At the moment webpack complains that some files are above the recommended size limit. It be interesting to see if there's a way to reduce this with some further  optimization eg lazyloading etc.. but I'd leave it for a subsequent PR for now
+
+```
+WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
+This can impact web performance.
+Assets: 
+  TimedTextEditor.js (271 KiB)
+  TranscriptEditor.js (455 KiB)
+  index.js (455 KiB)
+```
+
+
+## Excluding tests and sample json
+[webpack works out the dependency tree from the entry points](https://webpack.js.org/concepts/#entry) as it's making it's own [dependency graph](https://webpack.js.org/concepts/dependency-graph/), and because sample and tests files are not used or imported in production, then they are be automatically excluded in the bundle process.
+
+<!-- ## unrelated question s
+- [ ] npm script `fix-styles`? `prettier-stylelint` shall we add prettier for general code? -->
+
+## Jest config
+To run jest across the react components, we have to stub the css file dependencies using [`moduleNameMapper`](https://jestjs.io/docs/en/configuration#modulenamemapper-object-string-string) under the jest attribute in `package.json`
+
+
+## Storybook css modules
+- Storybook does not have support for css modules out of the box.
+- If you remove CRA scripts then [you get this issue](https://github.com/storybooks/storybook/issues/2320) where css modules don't load in storybook.
```

Comment:
What is CRA? Will other people know? I don't know.

Reply:
Good point, CRA  is [`create react app`](https://create-react-app.dev) - should add/link to it.